### ISSUE TITLE
Flutter accessability issues with iOS 16.1.2 and 16.1.1

### ISSUE THREAD

**Issue by drkhannah** [flutter/flutter#118321]
not sure if this is already reported but one of our customer has reported issues with flutter screen readers using iOS 16.1.2 and 16.1.1

Short summary of the functionality of the voiceover is that the client needs to swipe right to select items and navigation buttons displayed on the screen. Below are some things I observed in the call with the client:

- if you pause at all while navigating from the top of a given screen to the bottom of the screen and then try to continue moving through the screen, it moves the "selector" back to the top of the screen so they are effectively unable navigate through the app. You'd have to rapidly swipe to get to the bottom of the page and hope you land on the option you're looking for or at least near it to select what you're looking for.

- the navigation bar at the bottom of the app is also rendered more or less useless because of this issue

- because there is no "back" button on the portfolio page, clients using this feature are currently unable to navigate back to the "Home" page. This can make navigating between accounts difficult as the gestures change when using the Voiceover accessibility feature

Note: Client did say that it was working great prior to last iOS update about a week ago and was very happy with the new app until recently.

**Comment by darshankawar** [comment]
Thanks for the report @drkhannah 
There's a [similar issue](https://github.com/flutter/flutter/issues/113377) related to accessibility using iOS 16 version that you can take a look and see if it resembles your case or not. If yes, you may follow-up in that issue and share your findings in that for better tracking. 
If not, please provide a minimal code sample that showcases the behavior you described above, so that we'll have enough details to work with it.
